lib/docs/filters/padrino/clean_html.rb

```diff
@@ -0,0 +1,18 @@
+module Docs
+  class Padrino
+    class CleanHtmlFilter < Filter
+      def call
+        title = at_css('#filecontents h1')
+        @doc = at_css('#content')
+
+        css('.highlight').each do |node|
```

Comment:
What page is this for? I can't seem to find any `.highlight` element on the website.